pipelines/tasks/Kallisto.wdl

```diff
@@ -0,0 +1,155 @@
+
+task Mkref {
+  File transcriptome_fasta  # fasta file containing transcripts ONLY (NOT a full genome!)
+  Int k  # size of kmer in index; must be an odd number.
+
+  command {
+    kallisto index \
+      -i kallisto.idx \
+      -k ${k} \
+      "${transcriptome_fasta}"
+  }
+
+  runtime {
+    docker: "humancellatlas/kallisto:0.43.1"
+    cpu: 4
+    memory: "16 GB"
+    disks: "local-disk 150 HDD"
+  }
+
+  output {
+    File index = "kallisto.idx"
+  }
+}
+
+
+task QuantSingleEnd {
+  File r1  # forward read
+  File index  # kallisto index, output of Kallisto.Mkref task
+
+  # estimate disk requirements
+  Int disks = 10 + ceil(size(r1, "G") * 4 + size(index, "G") * 2)
+
+  command {
+    kallisto quant \
+      --index "${index}" \
+      --output-dir . \
+      --bootstrap-samples 100 \
+      --single \
+      --fragment-length 250 \
+      --sd 25 \
+      "${r1}"
+  }
+
+  runtime {
+    docker: "humancellatlas/kallisto:0.43.1"
+    cpu: 4
+    memory: "16 GB"
+    disks: "local-disk 100 HDD"
+  }
+
+  output {
+    File abundance_h5 = "abundance.h5"
+    File abundance_tsv = "abundance.tsv"
+    File log = "run_info.json"
+  }
+}
+
+
+task QuantPairedEnd {
+  File r1  # forward read
+  File index  # kallisto index, output of Kallisto.Mkref task
+  File r2  # reverse read
+
+  # estimate disk requirements
+  Int disks = 10 + ceil((size(r1, "G") + size(r2, "G")) * 4 + size(index, "G") * 2)
+
+  command {
+    kallisto quant \
```

Comment:
Can you add '--Pseudobam'  in cmd and output pseudoalignment bam file. cmd is something like 
kallisto quant -i index -o out --pseudobam r1.fastq r2.fastq | samtools view -Sb - > out.bam

Reply:
Finding alllll them undocumented requirements!

```
[quant] fragment length distribution will be estimated from the data
Error: pseudobam is not compatible with running on many threads.
```

Reply:
they should do better job on their documentation!!!